src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+    public String getRandomBall() {
+        int number = new Random().nextInt(100);
+        String color = ColorSupplier.getRandomColor();
+        super.setColor(color);
+        super.setNumber(number);
+        return color + " " + number;
+    }
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/63095184/be22ee59-c5b4-4b2c-8385-a7ac844ae4f8)
You need to return Ball, not String